src/app/_components/_sharedcomponents/Preferences/PreferencesSubElementVariants/AnimationOptionsTab.tsx

```diff
@@ -0,0 +1,343 @@
+import * as React from 'react';
+import { useState, useEffect, Dispatch, SetStateAction } from 'react';
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+import { Divider, Alert, RadioGroup, FormControlLabel, Radio, FormControl, ToggleButtonGroup, ToggleButton } from '@mui/material';
+import { useUser } from '@/app/_contexts/User.context';
+import {
+    loadPreferencesFromLocalStorage,
+} from '@/app/_utils/ServerAndLocalStorageUtils';
+import { IPreferences } from '@/app/_contexts/UserTypes';
+import PreferenceButton from '@/app/_components/_sharedcomponents/Preferences/_subComponents/PreferenceButton';
+import { keyframes } from '@mui/system';
+import { savePreferencesGeneric } from '@/app/_utils/genericPreferenceFunctions';
+
+interface AnimationOptionsTabProps {
+    setHasNewChanges?: Dispatch<SetStateAction<boolean>>;
+}
+
+enum SaveStatus {
+    NoChange = 'NoChange',
+    Success = 'Success',
+    Error = 'Error'
+}
+
+const pulseBorder = keyframes`
+  0% {
+    border-color: rgba(0, 140, 255, 0.4);
+    box-shadow: 0 0 4px rgba(0, 140, 255, 0.4);
+  }
+  50% {
+    border-color: rgba(0, 180, 255, 0.6);
+    box-shadow: 0 0 8px rgba(0, 180, 255, 0.6);
+  }
+  100% {
+    border-color: rgba(0, 140, 255, 0.4);
+    box-shadow: 0 0 4px rgba(0, 140, 255, 0.4);
+  }
+`;
```

Comment:
Are there any shared/global CSS or animations defined in this codebase? I see that this `pulseBorder` animation's code is duplicated for each settings page at least.

Standardizing and having a source of truth for styles is something that I'd be willing to take on sometime.